.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: rust
 
+branches:
+  only:
+  - master
+  - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
```

Comment:
I dunno if this a common thing, but reading such a regex puts me in trance a little bit :D
Shall we add a comment for this line maybe?

Reply:
What? You don't speak :camel:? :laughing: I guess `# release tags like v0.9, v0.9.2, v0.9.3-rc1` will be a good explanation with some examples.

Reply:
Well, it took me some time to decode that, but I don't "speak" it, yeah )

Reply:
I have added some comments. :upside_down_face: